_components/utilities/box-shadow.md

```diff
@@ -6,3 +6,116 @@ title: Box shadow
 category: Utilities
 lead: Set the box shadow
 ---
+
+<div class="size-sans-4 weight-300">
+
+  <h3 class="grid-col size-sans-micro text-normal text-medium text-uppercase text-ls-1 margin-y-0 margin-left-2px">CSS properties</h3>
+  <div class="margin-top-1">
+    <span class="token display-inline-block text-no-uppercase text-ls-auto padding-05 margin-top-05">box-shadow</span>
+  </div>
+
+  <section class="border-top-1px padding-top-1 margin-top-4">
+    <div class="grid-row flex-align-center margin-bottom-2">
+      <h2 class="grid-col-auto margin-0 text-light size-sans-8">Examples and usage</h2>
+      <p class="grid-col-fill text-right size-sans-1 text-normal margin-y-0 margin-left-2px text-gray-60">Utilities, values, and variants may be activated and deactivated in <a href="#0" class="text-text text-no-wrap">advanced settings</a>.</p>
+    </div>
+
+    <section class="text-color">
+      <section class="bg-white padding-2 radius-md margin-y-2 border-1px border-gray-10">
+        <section class="margin-x-neg-2 margin-bottom-2 padding-x-2 padding-bottom-105 border-bottom-1px border-gray-10">
+          <div class="grid-row">
+          <h3 class="grid-col-auto size-sans-6 margin-top-0 margin-bottom-105">Box shadow</h3>
+
+            <ul class="grid-col-fill text-right add-list-reset display-inline">
+              <li class="display-inline-block padding-y-05 padding-x-105 border-1px text-gray-20 radius-md margin-right-05 is-inverse text-strike">responsive</li>
+              <li class="display-inline-block padding-y-05 padding-x-105 border-1px text-gray-20 radius-md margin-right-05 is-inverse text-strike">active</li>
+              <li class="display-inline-block padding-y-05 padding-x-105 border-1px text-gray-20 radius-md margin-right-05 is-inverse text-strike">hover</li>
+              <li class="display-inline-block padding-y-05 padding-x-105 border-1px text-gray-20 radius-md margin-right-05 is-inverse text-strike">focus</li>
+              <li class="display-inline-block padding-y-05 padding-x-105 border-1px text-gray-20 radius-md margin-right-05 is-inverse text-strike">visited</li>
+            </ul>
+          </div>
+        </section>
+
+        <div class="grid-row">
+          <div class="grid-col-3 text-center display-flex flex-column flex-justify flex-align-start margin-bottom-2 border-bottom-1px border-gray-5 padding-bottom-1">
+            <div class="bg-secondary-light display-inline-block">
+              <div class="square-9 border-1px bg-white shadow-none"></div>
```

Comment:
I'd vote for removing the `border-1px` on this one just to avoid confusion, unless you want to add the border to all of the other boxes as well?

Reply:
If you don't add the border, you can't see the box and it looks invisible. Adding the border to the box shadow elements interferes with the appearance with the box shadow.

Reply:
I don't think it being invisible is a bad thing since that the expected behavior, but your call.

Reply:
That's true. So here's what it looks like without a border:
<img width="993" alt="screen shot 2018-05-31 at 11 17 34 am" src="https://user-images.githubusercontent.com/5249443/40800103-44d926c6-64c4-11e8-8a6d-5cb7c50be983.png">

Let's see what @thisisdano thinks.

Reply:
Not as bad as I thought it would be since it's in the context of the other items, but coud go either way.

Reply:
Updated in f790304.